Event-Framework-Tests.postman_collection.json

```diff
@@ -246,6 +246,859 @@
 				}
 			},
 			"response": []
+		},
+		{
+			"name": "Transfer Prepare Error - Invalid FSP",
+			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"id": "9d58aca2-4bdc-4c4b-b12b-b70c76f3c36d",
+						"exec": [
+							"var uuid = require('uuid');",
+							"var generatedUUID = uuid.v4();",
+							"",
+							"pm.environment.set('transfer_ID', generatedUUID);",
+							"",
+							"pm.variables.set('transferDate', (new Date()).toUTCString());",
+							"",
+							"pm.environment.set(\"transferExpiration\",new Date(new Date().getTime() + 600000));",
+							"",
+							"pm.variables.set('efk_url', 'http://dev1-elasticsearch.mojaloop.live/_all/_search?pretty');",
```

Comment:
The Efk URI should be generalised.

Reply:
EFK URI is generalised in local env file and other ENV files are updated accordingly.